src/NServiceBus.NHibernate/TimeoutPersisters/TimeoutEntity.cs

```diff
@@ -29,18 +29,26 @@ class TimeoutEntity
         public virtual byte[] State { get; set; }
 
         /// <summary>
+        /// Timeout endpoint name.
```

Comment:
Why do we need *both* the replacement in `OptimizedSchemaUpdated` and these comments? Would one of them not suffice?